lib/modules/stats/global.rb

```diff
@@ -42,6 +42,11 @@ def self.calculate_stats_for(klass, field_name)
     end
   end
 
+  def self.percentage_oecm_coverage
+    ((ProtectedArea.global_terrestrial_oecm_coverage 
+    + ProtectedArea.global_marine_oecm_coverage) / self.global_area).to_f * 100
```

Comment:
These are probably going to be reworked with the changes made by Luca in PR #500 . Ed has provided the OECM country stats which should be preferred for this kind of calculations

Reply:
Had a brief look at the new methods in the Stats::Global module - I will change to use those methods when that PR's been merged.

Reply:
I'm double checking how it's best to calculate the OECM only stats as there are multiple ways we could do that, but no idea which one should we use. I'll let you know when I get an answer